test_scripts/RC/MultipleModules/ModulesAllocation/061_GetInteriorVehicleDataConsent_all_correct_multiple_modules_rejected.lua

```diff
@@ -0,0 +1,114 @@
+--    SDL rejects allocation of modules (moduleType: <moduleType>, moduleId: <moduleId1>)
+--     and (moduleType: <moduleType>, moduleId: <moduleId2>) to App1
+--     and does not send OnRCStatus notifications to HMI and Apps
+--    SDL responds on SetInteriorVehicleData RPC with resultCode: REJECTED
+---------------------------------------------------------------------------------------------------
+local runner = require('user_modules/script_runner')
+local common = require("test_scripts/RC/MultipleModules/commonRCMulModules")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appLocation = {
+  [1] = common.grid.BACK_RIGHT_PASSENGER,
+  [2] = common.grid.BACK_CENTER_PASSENGER
+}
+
+local testServiceArea = common.grid.BACK_SEATS
+
+local rcAppIds = { 1, 2 }
+
+--[[ Local Functions ]]
+local function buildTestModulesStruct(pRcCapabilities)
+  local excludedModuleTypes = {"BUTTONS", "HMI_SETTINGS", "LIGHT"}
+  local modulesStuct = {}
+  for moduleType, modules in pairs(pRcCapabilities) do
+    if not common.isTableContains(excludedModuleTypes, moduleType) then
+      modulesStuct[moduleType] = {}
+      for _, rcModuleCapabilities in ipairs(modules) do
+        modulesStuct[moduleType][rcModuleCapabilities.moduleInfo.moduleId] = false
+      end
+    end
+  end
+  return modulesStuct
+end
+
+local rcCapabilities = common.initHmiRcCapabilitiesMultiConsent(testServiceArea)
+local testModules = buildTestModulesStruct(rcCapabilities)
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Prepare preloaded policy table", common.preparePreloadedPT, { rcAppIds })
+runner.Step("Prepare RC modules capabilities and initial modules data", common.initHmiDataState, { rcCapabilities })
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { rcCapabilities })
+runner.Step("Set RA mode: ASK_DRIVER", common.defineRAMode, { true, "ASK_DRIVER" })
+runner.Step("Register App1", common.registerAppWOPTU, { 1 })
+runner.Step("Register App2", common.registerAppWOPTU, { 2 })
+runner.Step("Activate App1", common.activateApp, { 1 })
+runner.Step("Send user location of App1 (Back Seat)", common.setUserLocation, { 1, appLocation[1] })
+runner.Step("Activate App2", common.activateApp, { 2 })
+runner.Step("Send user location of App2 (Back seat)", common.setUserLocation, { 2, appLocation[2] })
+
+for moduleType, consentArray in pairs(testModules) do
+  for moduleId in pairs(consentArray) do
+    runner.Step("Allocate module [" .. moduleType .. ":" .. moduleId .. "] to App2",
+      common.rpcSuccess, { moduleType, moduleId, 2, "SetInteriorVehicleData" })
+  end
+end
+
+runner.Title("Test")
+runner.Step("Activate App1", common.activateApp, { 1 })
+for moduleType, consentArray in pairs(testModules) do
+  runner.Step("Disallow " .. moduleType .. " modules reallocation to App1 without asking driver",
```

Comment:
This step does get negative consent from driver despite saying `without asking driver`.